src/index.html

```diff
@@ -2,13 +2,50 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="./images/logo.png" alt="logotip"
+        class="logo__image">
+      </a>
+      <nav class="navbar">
+        <ul class="nav__list">
+          <li class="nav__text">
```

Comment:
Add empty lines between multiline sibling blocks of HTM
<img width="1106" alt="Screenshot 2023-10-09 at 14 20 38" src="https://github.com/mate-academy/layout_moyo-header/assets/64746943/743f8b74-4e29-4506-ae71-cb355fbe16c2">